edgelet/iotedge/src/check/mod.rs

```diff
@@ -856,18 +861,56 @@ mod tests {
     #[test]
     fn settings_connection_string_dps_err() {
```

Comment:
It's not an `_err` test any more. Maybe add a new `_err` test for when the backup isn't there?